test/parallel/test-async-wrap-check-providers.js

```diff
@@ -1,6 +1,11 @@
 'use strict';
 
 const common = require('../common');
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
+
```

Comment:
Hm, this test is basically laid out to use *all* providers, so I guess that would really require its own test file… I’m not sure that’s worth the trouble.